config/runtimes/vllm-gaudi-template.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+  annotations:
+    description: vLLM ServingRuntime to support Gaudi(for Habana AI processors)
+    openshift.io/display-name: vLLM Gaudi ServingRuntime for KServe
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/vllm
+    template.openshift.io/long-description: This template defines resources needed to deploy vLLM ServingRuntime with KServe in Red Hat OpenShift AI
+    opendatahub.io/modelServingSupport: '["single"]'
+    opendatahub.io/apiProtocol: 'REST'
+  name: vllm-gaudi-runtime-template
+objects:
+  - apiVersion: serving.kserve.io/v1alpha1
+    kind: ServingRuntime
+    metadata:
+      name: vllm-gaudi-runtime
+      annotations:
+        openshift.io/display-name: vLLM Gaudi ServingRuntime for KServe
```

Comment:
Could `vLLM ServingRuntime with Gaudi accelerators support for KServe` be more accurate? IDK if the name has been reviewed, nor know if this string would be displayed in the dashboard. So, not sure if accuracy is really relevant.

Reply:
I had updated the display name with `vLLM ServingRuntime with Gaudi accelerators support for KServe`